tools/stubmaker/main.go

```diff
@@ -225,21 +228,23 @@ func isStubNeeded(funcs []string) (bool, error) {
 	case len(found) == len(funcs):
 		return false, nil
 	case len(found) != 0:
-		return false, fmt.Errorf("funcs partially defined: need=%v, found=%v", funcs, found)
+		sort.Strings(found)
+		sort.Strings(funcs)
+		delta := cmp.Diff(found, funcs)
+		return false, fmt.Errorf("funcs partially defined, delta=%s", delta)
 	}
 
 	return true, nil
 }
 
+var funcRE = regexp.MustCompile("^func *[(][^)]+[)] *([^(]+)")
```

Comment:
Is this regex supposed to match both functions and methods? It looks like it doesn't match functions, as it's written now

Reply:
Good catch, ty.  The code evolved as I wrote it and I forgot about the original use case.